### PR TITLE
docs: Add guide on contributing a code patch

### LINKED ISSUE #8035: [DOC] Notes and minor errors in the doc
## Description
1) On this [page](https://docs.django-cms.org/en/latest/contributing/code.html),  
>```
> *************
> In a nutshell
> *************
> 
> Here's what the contribution process looks like in brief:
> 
> #. Fork our `GitHub`_ repository, https://github.com/django-cms/django-cms
> #. Work locally and push your changes to your repository.
> ```
I thought "Work locally and push your changes to your repository" is quite ambiguous,
especially for a newbie. One idea that comes to mind would be to link to a section that
talks about how to setup djangoCMS from git source. Perhaps there is a link to it already too

I noticed these minor doc errors while trying to install django-cms.

2) I think [here](https://docs.django-cms.org/en/latest/introduction/01-install.html#install-django-cms-by-hand:~:text=The%20last%20option%20is%20to%20install%20is%20creating%20a%20django%20CMS%20project%20manually%20into%20a%20local%20virtual%20environment%20by%20running%20the%20djangocms%20command) could be reworded but the issue can be closed if it's too minor. It could be reworded to
`The last option to install is creating a django CMS project
manually into a local virtual environment by running the 
djangocms command` (removing the first `is`, so it reads better.)

3) On the same page, under `Installing Django CMS by hand` section.
```
.. note::

    You can also use this guide to help you install django CMS as part of an existing
    project. However, the guide assumes that you are starting with a blank project, so
    you will need to adapt the steps below appropriately as required. You
```
I think the "You" that ends the paragraph can be removed, unless the writer is continuing a thought.

## Do you want to help fix this documentation issue?
* [x] Yes, I want to help fix this issue and I will join the channel #documentation on our [Discord Server](https://discord-docs-channel.django-cms.org) to confirm with the team that a PR is welcome.
* [ ] No, I only want to report the issue.

### PR DESCRIPTION
**Pull request by mrbazzan**
* add guide on contributing code patch
* fix minor typos

## Description
Fixes #8035 

## Checklist

* [x] I have opened this pull request against ``develop-4``
* [ ] I have added or modified the tests when changing logic
* [ ] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [ ] I have read the [contribution guidelines ](https://github.com/django-cms/django-cms/blob/develop/CONTRIBUTING.rst) and I have joined the channel #pr-reviews on our [Discord Server](https://discord-pr-review-channel.django-cms.org) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

**Review by fsbraun — COMMENTED** [review]
I've got just a tiny comment on this.

File: docs/contributing/contribute-patch.rst [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+#. Fork the `django CMS project <https://github.com/django-cms/django-cms>`_ GitHub repository to your
+   own GitHub account
+#. Clone your fork locally::
+
+    git clone git@github.com:YOUR_USERNAME/django-cms.git
+
+#. Create a virtualenv::
+
+    python3 -m venv .venv
```
[the remark this answers is not in the document]
Comment (marksweb) [reply docs/contributing/contribute-patch.rst:23]: Agreed.

You'd `cd` after a clone, then create the venv in the project directory or at a location you keep all venvs like a `~/.virtualenvs`. But if someone has that setup they'll know how they want this and do that step their way.

[the remark this answers is not in the document]
Comment (mrbazzan) [reply docs/contributing/contribute-patch.rst:23]: Totally agree!

**Review by fsbraun — APPROVED** [review]
Great work, @mrbazzan !

**Pull request merged**